mu-plugins/blocks/time/postcss/style.pcss

```diff
@@ -0,0 +1,3 @@
+.wporg-time {
+	text-decoration: underline;
```

Comment:
I find it's useful to be able to see the content the format has been applied to, and this seems the most appropriate style given that it will end up as a link on the frontend.